website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,98 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### LDAP Auth Method Attributes
+
+The ldap auth method has the following additional attributes:
+
+- `state` - The state of the auth method; either inactive, active-private or
+  active-public
+
+- `start_tls` - (optional) If true, issues a StartTLS command after establishing
+  an unencrypted connection. Defaults to false.
+
+- `insecure_tls` - (optional) If true, skips LDAP server SSL certificate
+  validation - which is insecure and should be used with caution. Defaults to
```

Comment:
I feel a comma is more appropriate here, what do you think?
```suggestion
  validation, which is insecure and should be used with caution. Defaults to
```

Reply:
ty!